src/components/meetups/FilterGenre.jsx

```diff
@@ -0,0 +1,36 @@
+function FilterGenre({ onChange, selectedGenre }) {
+  const allGenres = [
```

Comment:
Nice work! How did you get the list?

Reply:
Thank you!
 I did on the console of the browser. I take the JSON file from the Code Editor to console and there I use map and forEach to take all the genres. 

x.map((item) => item.genre).flat().forEach((genre) => allGenres[genre] = genre) 
JSON.stringify(Object.keys(allGenres))

And the magic happened :D